go/externals/social_assertion.go

```diff
@@ -24,3 +24,11 @@ func AssertionParseAndOnly(s string) (libkb.AssertionExpression, error) {
 func AssertionParse(s string) (libkb.AssertionExpression, error) {
 	return libkb.AssertionParse(MakeAssertionContext(), s)
 }
+
+func ParseAssertionsWithReaders(s string) (writers, readers []libkb.AssertionExpression, err error) {
+	return libkb.ParseAssertionsWithReaders(MakeAssertionContext(), s)
+}
+
+func ParseAssertionList(s string) ([]libkb.AssertionExpression, error) {
+	return libkb.ParseAssertionList(MakeAssertionContext(), s)
+}
```

Comment:
Maybe some unit tests around this?   Depending on what coverage you have for the different parsing possibilities.  I find with parsing stuff unit tests can be helpful.

Reply:
added TestParseAssertionsWithReaders